jwt.go

```diff
@@ -0,0 +1,67 @@
+package clerk
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-jose/go-jose/v3/jwt"
+)
+
+type key string
+
+const clerkActiveSessionClaims = key("clerkActiveSessionClaims")
+
+// ContextWithSessionClaims returns a new context which includes the
+// active session claims.
+func ContextWithSessionClaims(ctx context.Context, value any) context.Context {
```

Comment:
🔧 Shouldn't the `value` type here be `*SessionClaims` 🤔 ?

Reply:
:thinking: I'm not sure about that. TBD after we settle on custom type for the claims.